api/Event.json

```
@@ -967,10 +967,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "3"
```

This should also be Safari 1 based on https://github.com/WebKit/WebKit/blob/b923c2f4eb7804aae65d67b4c3061d6435acb10d/WebCore/khtml/ecma/kjs_events.cpp#L188

```suggestion
              "version_added": "1"
```